Mathlib/Order/Circular.lean

```diff
@@ -402,6 +402,17 @@ abbrev LinearOrder.toCircularOrder (α : Type*) [LinearOrder α] : CircularOrder
       · exact Or.inr (Or.inl ⟨hcb, hba⟩)
       · exact Or.inr (Or.inr <| Or.inl ⟨hba, hac⟩) }
 
+section
+
+variable {α : Type*} [LinearOrder α] {a b c : α}
+
+attribute [local instance] LinearOrder.toCircularOrder
+
+lemma sbtw_iff : sbtw a b c ↔ a < b ∧ b < c ∨ b < c ∧ c < a ∨ c < a ∧ a < b := .rfl
+lemma btw_iff : btw a b c ↔ a ≤ b ∧ b ≤ c ∨ b ≤ c ∧ c ≤ a ∨ c ≤ a ∧ a ≤ b := .rfl
```

Comment:
These should probably mention in their docstring that they are about the non-instance `LinearOrder.toCircularOrder`